napi/oxlint2/src-js/index.ts

```diff
@@ -36,8 +36,12 @@ interface DiagnosticReport {
   ruleIndex: number;
 }
 
+export type Node = Object;
```

Comment:
😬

Reply:
I guess this could be `@oxc-project/types`'s `Node` type? 
https://www.npmjs.com/package/@oxc-project/types?activeTab=code (last code block)

Alternative, If you want it very strict:
```
export type Node = {
  [k: string]: unknown
}
```